src/util.rs

```diff
@@ -271,6 +271,25 @@ pub fn compare_versions(old: &str, new: &str) -> Option<bool> {
     }
 }
 
+/// Returns the value of the WAPM_RUNTIME env var if it exists.
+/// Otherwise returns wasmer
+pub fn get_runtime() -> String {
```

Comment:
I think it might be better to have `get_runtime` return the tuple, it's not obvious that the `WAPM_RUNTIME` can env var can be set to something which may not work as a runtime (unless it will even with args, then nevermind).

What I mean is that by splitting it here it prevents incorrect usage!